src/Company/Company.php

```diff
@@ -119,6 +124,7 @@ public function __construct(array $data)
         $this->legalRepresentativeFirstName = to_string($data['legalRepresentativeFirstName']);
         $this->legalRepresentativeLastName = to_string($data['legalRepresentativeLastName']);
         $this->extra = (array) $data['extra'];
+        $this->nafCode = (isset($data['nafCode']) && !is_null($data['nafCode']) ? to_string($data['nafCode']) : "");
```

Comment:
```suggestion
        $this->nafCode = array_key_exist('nafCode', $data) ? to_string($data['nafCode']) : null;
```

Si tu dis que ta variable c'est soit null soit une string, il faut être cohérent avec ce que tu mets dans le __construct

Reply:
j'avais mis isset car c'est ce qui est utilisé pour faire le même type de contrôle plus haut, mais effectivement le résultat est le même tout en étant plus lisible avec array_key_exist, je corrige